### DISCUSSION TITLE
Multi step forecasting (univariate/multivariate)

### DISCUSSION THREAD

**Discussion by mandar-avhad** [timeseriesAI/tsai#438] [General]
Hi, does tsai has support for multi-step forecasting technique?
If yes, can you please share the notebook or reference for it?

**Comment by oguiza** [comment]
Hi @mandar-avhad,
Unfortunately, there's no tutorial notebook yet. 
Please, take a look at this [example](https://github.com/timeseriesAI/tsai#multi-step). The key really is to prepare the input (X and y). You can learn more about data preparation [here](https://timeseriesai.github.io/tsai/data.preparation.html).

**Reply by mandar-avhad** [reply]
Sure, thanks.
But this is for univariate forecasting. Do we support multi-step for multi-variate forecasting?

**Reply by oguiza** [reply]
> Do we support multi-step for multi-variate forecasting?

Yes. It works exactly the same way. Just prepare your y with shape [n_samples x n_vars x n_steps]. It would work without further changes. If you use it, please, let me know how it works.
